src/main/haskell/kore/src/Kore/Attribute/Parser.hs

```diff
@@ -102,7 +120,11 @@ data ParseError
 newtype Parser a =
     Parser
     { getParser :: ReaderT AttributeMap (Either (Error ParseError)) a }
-  deriving (Applicative, Functor, Monad)
+  deriving (Functor, Applicative, Monad)
+
+instance Alternative Parser where
+    empty = throwError (Kore.Error.koreError "Kore.Attribute.Parser.empty")
```

Comment:
Reaching this code seems to be a user error and, until now, we used `error` for these cases.

Reply:
No, this sort of error is what parser functions like `choose` use to backtrack and try an alternative parse. `empty` is supposed to be a unit of `choose`.